examples/load-gen.ts

```diff
@@ -414,11 +455,10 @@ const loadGeneratorOptions: BasicJavaScriptLoadGenOptions = {
    */
   numberOfConcurrentRequests: 10,
   /**
-   * Controls how long the load test will run.  We will execute this many operations
-   * (1 cache 'set' followed immediately by 1 'get') across all of our concurrent
-   * workers before exiting.  Statistics will be logged every 1000 operations.
+   * Controls how long the load test will run, in milliseconds. We will execute operations
+   * for this long and the exit. The default is 60 seconds.
    */
-  totalNumberOfOperationsToExecute: 50_000,
+  totalMsToRun: 60 * 1000,
```

Comment:
same nit here, i don't think anyone is ever going to target ms granularity for the duration; seconds would probably be fine?

Reply:
I'm avoiding having some be in seconds and some be in milliseconds. If I change one, I'll change them all.